drafts/WEP0001.rst

```diff
@@ -0,0 +1,273 @@
+
+Proposal
+========
+
+Overview
+--------
+
+Write the Docs conferences can offer grants to attendees and speakers.
+The possibilities depend on the available budget, which may be different
+per conference.
+
+Some important considerations apply to grants:
+
+* There are generally more applications than funds available, so there must
+  be a process to prioritise grant allocation.
+* People can be reluctant, or even feel shameful about applying, so the process
+  for applications should be encouraging and welcoming, even for people who
+  feel they might not deserve it.
+* Grants improve access for marginalised and underrepresented groups already,
+  but an option can be to prioritise marginalised groups in allocations.
+  Currently, this proposal is written with prioritisation of marginalised and
+  underrepresented groups.
+
+Speakers
+--------
+
+Accepted speakers are an essential part of the conference, and should
+therefore have a special grant priority. All speakers and organisers will
+receive a free ticket, as they already pay for their participation in the
+conference with their volunteer time.
+
+Either at the time of the CFP (in which case it should be stressed on this
+form that choices will not influence talk selection) or later, all speakers
+will be asked whether (pick one option):
+
+* They are willing and able to pay their own costs completely.
+* Their employer is funding their travel and accommodation.
+* They would like to request a (partial) grant (if so, we need their details -
+  a partial grant could mean "I need you to cover my flight, but I can arrange
+  my own accommodation".)
+
+If their employer is funding their attendance, the employer will be recognised
+as a low level sponsor (perhaps a bit higher in case of very expensive travel).
+This may help people to get funding from their employer. This is reasonable,
+as the employer is paying for a speaker's ability contribute. The CFP 
+should make clear that we have this sponsor recognition policy, as it can help 
+people talk to and convince their employers to fund the trip earlier.
+
+Grants for speakers follow the regular process for assignment and handling,
+except that they are prioritised all other grant applications.
+
+Regular grant requests
+----------------------
+
+(Prospective) attendees can request a grant through an application form.
+Other than typical details like their name (both legal and preferred),
+and contact details, the form should ask roughly these questions:
+
+* What they do with documentation
+* Why they want to attend Write the Docs
+* An estimate for their costs (provide tips for looking on booking, airbnb,
+  google flights and possibly others), split out by cost item.
+  Travel and accommodation are typical costs, but applicants may specify
+  and ask for any costs they need covered to be able to attend.
+* Which country they live in
+* Whether they are part of an underrepresented or marginalised group in tech
+
```

Comment:
Shall the form also guarantee adherence to Write the Docs code of conduct?

Reply:
Yes, that's a good addition, I'll add that.

Reply:
Actually, I'm not sure we need that. They'd get special links to register their ticket - all staff does as well - and the registration form has this question already. Feels a bit superfluous to ask twice.

Reply:
I think it's probably worth mentioning the COC in the grant copy to set expectations, but yea, I don't know if we need a formal checkmark that they agree to it, since that would come with the ticket registration.

Reply:
I have added a note of this to our draft application form.